### PR TITLE
Admin - Order management

### PR DESCRIPTION
**Pull request by hduong1207uet**
## Related Tickets:
- [#35060](https://edu-redmine.sun-asterisk.vn/issues/35060)

## WHAT this PR does?
Admin - Order management

## Notes Impacted Areas:
- N/A.

## Attachments:
- image
- image

### CODE REVIEW

**Comment by hduong1207uet** [comment]
ready

File: app/Http/Controllers/Admin/OrderController.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Admin_OrderFormRequest $request)
+    {
+        Order::create($request->all());
+
+        return redirect(route('orders.index'))->with('success', __('order_created'));
+    }
```
Comment (congnx-2423) [inline app/Http/Controllers/Admin/OrderController.php]: trường hợp fail thì sao nhỉ

File: app/Http/Controllers/Admin/OrderController.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $userIds = User::select('id', 'name', 'phone_number')->get();
```
Comment (congnx-2423) [inline app/Http/Controllers/Admin/OrderController.php]: đặt lại tên biến nhé
```suggestion
        $users = User::select('id', 'name', 'phone_number')->get();        
```

File: app/Http/Controllers/Admin/OrderController.php [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        $userIds = User::select('id', 'name', 'phone_number')->get();        
+
+        return view('admin.orders.create', compact('userIds'));
```
Comment (congnx-2423) [inline app/Http/Controllers/Admin/OrderController.php]:
```suggestion
        return view('admin.orders.create', compact('users'));
```

File: app/Http/Controllers/Admin/OrderController.php [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $order = Order::findOrFail($id);
+        $userIds = User::select('id', 'name', 'phone_number')->get();
+
+        return view('admin.orders.edit', compact('order', 'userIds'));
+    }
```
Comment (congnx-2423) [inline app/Http/Controllers/Admin/OrderController.php]:
```suggestion
    public function edit($id)
    {
        $order = Order::findOrFail($id);
        $users = User::select('id', 'name', 'phone_number')->get();

        return view('admin.orders.edit', compact('order', 'users'));
    }
```

**Review by ghost — APPROVED** [review]

**Comment by hduong1207uet** [comment]
ready

**Comment by hduong1207uet** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Review by huyenttm-0901 — COMMENTED** [review]
conflict

**Comment by hduong1207uet** [comment]
ready

**Review by congnx-2423 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**